app/views/api/v2/hosts/main.json.rabl

```diff
@@ -10,7 +10,7 @@ extends "api/v2/hosts/base"
 attributes :ip, :environment_id, :environment_name, :last_report, :mac, :realm_id, :realm_name,
            :sp_mac, :sp_ip, :sp_name, :domain_id, :domain_name, :architecture_id, :architecture_name, :operatingsystem_id, :operatingsystem_name,
            :subnet_id, :subnet_name, :sp_subnet_id, :ptable_id, :ptable_name, :medium_id, :medium_name, :build,
-           :comment, :disk, :installed_at, :model_id, :hostgroup_id, :hostgroup_name, :owner_id, :owner_type,
```

Comment:
@dLobatog if she added it back, meaning added an extra attribute for hostgroup_title I'm fine with it, I am not following why simply not add hostgroup_title and thats it? is it to support the condition? I think also responding with hostgroup_name:null is fine?

as a side note,  as a API consumer, I'm not sure I like this API output all together, and I would be very much prefer a better output similar to http://jsonapi.org/ instead of mix and match of object properties inside another (inside host response, we have hostgroup_id, hostgroup_name, hostgroup_title etc, we might as well return the hostgroup object in a separate response similar to rails eager loading)